packages/glibc/tz-utc.txt

```diff
@@ -0,0 +1 @@
+VFppZjIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAQAAAAAAABVVEMAVFppZjIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAQAAAAAAABVVEMAClVUQzAK
```

Comment:
Is there a reason to have this base64 encoded? It looks like the actual plain text content is smaller than its encoded value, plus wouldn't need the extra processing in the rpm creation.

Reply:
It's a binary file, and I'd like to avoid checking it into Git.